### ISSUE TITLE
FlowCharting Details not visible

### ISSUE THREAD

**Issue by 5up4D4v3** [algenty/grafana-flowcharting#197]
Hi,

I just installed the FlowCharting plugin into my Grafana container. 

Log:
![image](https://user-images.githubusercontent.com/54169608/85526358-3ac2e200-b60a-11ea-9e8e-04781863b9a9.png)

Grafana:
The plugin is installed (Configuration/Plugins):
![image](https://user-images.githubusercontent.com/54169608/85527002-c472af80-b60a-11ea-9538-f3b062952c54.png)

Then I create a new dashboard and add a new panel. For visualization I have the option for FlowCharting. If I choose it, the upper and below parts are completely empty:
![image](https://user-images.githubusercontent.com/54169608/85527381-1d424800-b60b-11ea-92c0-2798c0c23369.png)

In other visualizations there are options:
![image](https://user-images.githubusercontent.com/54169608/85527521-3cd97080-b60b-11ea-9e3a-bc75c92969c7.png)

What is wrong?

I use Grafana v6.4.4 and FlowCharting 0.9.0

Thanks for your help,
David

**Comment by algenty** [comment]
Have you a stack trace in console ?
What is your browser ?
Disable Ad blocker extension if you have it.

**Comment by 5up4D4v3** [comment]
There are no errors in the log file.
Same behaviour with Firefox 77.0.1 and Chrome 79.0, both 64-Bit and without any extension.

**Comment by algenty** [comment]
Can you try to save it et open it again ?
You can try also resize screen to force refresh
I can't reproduce this case
Can you configuration in edit mode ?

**Comment by 5up4D4v3** [comment]
I tried to reopen the browser, I tried to restart my container, I tried to resize the screen.
> Can you configuration in edit mode ?
What do you mean? The configuration of the dashboard?
![image](https://user-images.githubusercontent.com/54169608/85697900-7b8a2c00-b6da-11ea-99c5-0db31c6200fc.png)

**Comment by kboonmee** [comment]
See same issue, Grafana 6.3.5, FlowCharting 0.9.0 , Browser Safari 13.11

**Comment by algenty** [comment]
I can't reproduce it, are you able to test an alpha version to resolve it ?

**Comment by 5up4D4v3** [comment]
> I can't reproduce it, are you able to test an alpha version to resolve it ?

I can test, but only via my config map:
GF_INSTALL_PLUGINS 	agenty-flowcharting-panel

I cannot install via zipfile.

**Comment by algenty** [comment]
Hi, if possible, can you try version 0.9.1 downloadable at 
https://algenty.github.io/flowcharting-repository/ARCHIVES.html

**Comment by algenty** [comment]
any news ?

**Comment by kboonmee** [comment]
Unfortunely, version 0.9.1 not fix. more information I upgrade from version 0.6.0 to 0.9.0

**Comment by Matt-labor** [comment]
> Unfortunely, version 0.9.1 not fix. more information I upgrade from version 0.6.0 to 0.9.0
> 0.6.0 ok
> 0.7.1 not visible
> 0.8.1 not visible

Do you have any new progress or you just went back to 0.6.0

**Comment by samgrigg** [comment]
Hello!
I too am seeing this issue. I'm using Grafana version 6.4.3 and saw this behavior in versions 0.9.0 and 0.9.1 of FlowCharting. After trying a few more versions, I was able to get 0.8.0 working, but not 0.8.1. So I looked through some of the commits between those versions and may have found a clue. After seeing the following stack trace:

```Plugin component error TypeError: Cannot read property 'render' of undefined
    at e (module.js:2)
    at Object.invoke (angular.js:5040)
    at M.instance (angular.js:11000)
    at it (angular.js:9852)
    at angular.js:9174
    at angular.js:9039
    at variableQueryEditorLoader.tsx:23
    at d (ChangeTracker.ts:14)
    at ChangeTracker.ts:25```

Looking at the Grafana docs (https://grafana.com/docs/grafana/latest/packages_api/data/panelevents/), the PanelEvents namespace is listed as being in developer preview, so I wonder if it's not availalbe in Grafana 6.*?

I monkey-patched module.js, reverting the `PanelEvents.*` changes back to using the string event names. After that the plugin settings controls loaded.

Hope this is helpful!!
